apps/src/lib/ui/ParentLetter.jsx

```diff
@@ -149,8 +149,8 @@ class ParentLetter extends React.Component {
           <p>
             Code.org assigns utmost importance to student safety and security.
             Code.org has signed the{' '}
-            <a href={PRIVACY_PLEDGE_URL}>Student Privacy Pledge</a> and their{' '}
-            <a href={pegasus('/privacy')}>privacy practices</a> have received{' '}
+            <a href={PRIVACY_PLEDGE_URL}>Student Privacy Pledge</a> and their
+            privacy practices have received{' '}
```

Comment:
Whoops! This change, removing one of the privacy links, was supposed to be in the last PR but I forgot to push it.  Removed with Amanda's approval because it's basically redundant with other links in the privacy paragraph.